src/core/Akka/Actor/Stash/IWithBoundedStash.cs

```diff
@@ -17,8 +17,8 @@ namespace Akka.Actor
     /// <code>public IStash Stash { get; set; }</code>
     /// </summary>
     // ReSharper disable once InconsistentNaming
-    [Obsolete("Bounded stashing is not yet implemented. Unbounded stashing will be used instead [0.7.0]")]
-    public interface IWithBoundedStash : IActorStash, IRequiresMessageQueue<IBoundedDequeBasedMessageQueueSemantics>
+    [Obsolete("Use `IWithStash` with a configured BoundedDeque-based mailbox instead.")]
+    public interface IWithBoundedStash : IWithUnrestrictedStash, IRequiresMessageQueue<IBoundedDequeBasedMessageQueueSemantics>
```

Comment:
We can remove this type, since typically you'd want to change the mailbox or stash capacity with a bounded deque, which cannot be done programmatically (AFAIK)

Reply:
Let's remove this type in v1.5 in a separate PR - I'd like to backport this as-is to v1.4 first.